.drone.yml

```diff
@@ -0,0 +1,18 @@
+build:
+  image: gregway/gbm_immune_validation:latest
```

Comment:
So this uses the latest `gregway/gbm_immune_validation` on Docker hub. Do you have to manually upload to Docker hub?